gitlab-runner-17.1.yaml

```diff
@@ -27,15 +30,22 @@ pipeline:
     with:
       deps: github.com/Azure/azure-sdk-for-go/sdk/azidentity@v1.6.0 github.com/hashicorp/go-retryablehttp@v0.7.7
 
+  - runs: |
+      mkdir -p "${{targets.destdir}}"/
+      cp dockerfiles/runner/apline/entrypoint "${{targets.destdir}}"/entrypoint
+
   - uses: go/build
     with:
       packages: .
       output: gitlab-runner
       ldflags: -s -w -X gitlab.com/gitlab-org/gitlab-runner/common.NAME=${{package.name}} -X gitlab.com/gitlab-org/gitlab-runner/common.VERSION=v${{package.version}}
 
 subpackages:
-  - name: gitlab-runner-helper
+  - name: ${{package.name}}-helper
```

Comment:
I don't think that you could use package.name here anymore because then it will end up having gitlab-runner-17.1-helper but it should be gitlab-runner-helper-17.1 in this case

Reply:
So we just hardcode it, in that case? All right